src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder.cs

```diff
@@ -642,14 +520,14 @@ bool metadataTypeMatches(SymbolSet s, INamedTypeSymbol t)
             }
         }
 
-        private static bool TypeDerivesFrom(
-            SymbolSet metadataTypes, INamedTypeSymbol type, bool transitive)
+        private static bool TypeHasBaseTypeInSet(
+            SymbolSet set, INamedTypeSymbol type, bool transitive)
```

Comment:
renamed these for clarity.  'DerivesFrom' isn't really clear (esp. when interfaces are involved).